payments/mobile-pay/capture.md

```diff
@@ -0,0 +1,17 @@
+title: MobilePay Payments – Capture
```

Comment:
Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)